### DISCUSSION TITLE
DÚVIDA - Começando com Angular

### DISCUSSION THREAD

**Discussion by pedrovazfontes** [balta-io/forum#200] [Tira Dúvidas] [answered]
Pessoal, boa tarde!
Feliz ano novo a todos
Na aula Módulos e Componentes o professor mostra o arquivo main.ts no começo do vídeo..
Entretanto na minha maquina aparece um tanto quanto diferente...
Não aparece aquelas linhas de do promode.
Vou anexar uma imagem.
é alguma config diferente? ou é por conta da versão que é nova?
instalei a currente
![print](https://user-images.githubusercontent.com/61070351/210183571-ae0bebc9-8a31-4444-ade9-0f250c142952.png)

**Comment by andrebaltieri** [comment, marked as answer]
Boa noite, @pedrovazfontes, como vai?
Feliz ano novo 🚀

Suas versões estão iguais as do curso? Precisam ser as mesmas!
